llvm/include/llvm/MC/DXContainerPSVInfo.h

```diff
@@ -65,18 +68,97 @@ struct PSVRuntimeInfo {
   SmallVector<uint32_t> InputPatchMap;
   SmallVector<uint32_t> PatchOutputMap;
 
+  StringTableBuilder DXConStrTabBuilder;
```

Comment:
Any fields that shouldn't be set by the external caller we should make private. We should also look at these to see if they need to be members or if they should just move to the `finalize` function instead of being in the `write` function.

Reply:
Ahh.. Good point.  I'll go over this again and see what I can do.